pom.xml

```diff
@@ -15,7 +15,7 @@
     </modules>
 
     <properties>
-        <revision>8.0.1</revision>
+        <revision>8.1.0</revision>
```

Comment:
I think it is rather unlikely that we will do such a release.

Reply:
What version would you suggest?

Reply:
A major version increment, but we might also do that later, when we have more details on the next release.